org.mixedrealitytoolkit.core/Tests/TestUtilities/PackageValidatorResults.cs

```diff
@@ -0,0 +1,216 @@
+    /// </summary>
+    public class PackageValidatorResults
+    {
+        /// <summary>
+        /// The type of message.
+        /// </summary>
+        [Flags]
+        public enum MessageType
+        {
+            /// <summary>
+            /// A failed message.
+            /// </summary>
+            Failed = 1,
+
+            /// <summary>
+            /// A succeeded message.
+            /// </summary>
+            Succeeded = 2,
+
+            /// <summary>
+            /// A message for a test that was not run.
+            /// </summary>
+            NotRun = 4
+        }
+
+        private string fileName;
+        private string directoryPath = "Library\\ASValidationSuiteResults";
+
+        /// <summary>
+        /// The number of failed tests.
+        /// </summary>
+        public int FailedCount { get; private set; }
+
+        /// <summary>
+        /// The number of succeeded tests.
+        /// </summary>
+        public int SucceededCount { get; private set; }
+
+        /// <summary>
+        /// The number of tests that were not run.
+        /// </summary>
+        /// <value></value>
+        public int NotRunCount { get; private set; }
+
+        /// <summary>
+        /// The list of failed messages.
+        /// </summary>
+        public List<string> FailedMessages { get; private set; } = new List<string>();
+
+        /// <summary>
+        /// The list of succeeded messages.
+        /// </summary>
+        public List<string> SucceededMessages { get; private set; } = new List<string>();
+
+        /// <summary>
+        /// The list of not run messages.
+        /// </summary>
+        public List<string> NotRunMessages { get; private set; } = new List<string>();
+
+        /// <summary>
+        /// Get or set whether to ignore account related errors.
+        /// </summary>
+        public bool IgnoreAccountErrors { get; set; } = true;
+
+
+        /// <summary>
+        /// Create a new instance of the PackageValidatorResults class.
+        /// </summary>
+        /// <param name="packageId">
+        /// The package ID who's validation results will be loaded.
+        /// </param>
+        public PackageValidatorResults(string packageId)
+        {
+            fileName = packageId + ".txt";
+        }
+
+        /// <summary>
+        /// Load the results from the package validation log.
+        /// </summary>
+        /// <exception cref="FileNotFoundException">
+        /// Thrown if the package's log file is not found.
+        /// </exception>
+        public void LoadResults()
+        {
+            string filePath = Path.Combine(Application.dataPath, "..", directoryPath, fileName);
+            if (!File.Exists(filePath))
+            {
+                throw new FileNotFoundException($"File not found: {filePath}");
+            }
+
+            string fileContent = File.ReadAllText(filePath);
+            var startIndex = fileContent.IndexOf("VALIDATION RESULTS:");
```

Comment:
`"VALIDATION RESULTS:"` is repeated throughout the code, better place it in a class-level `const string ValidationResultsCaption="VALIDATION RESULTS:"` or something similar, to facilitate conde maintenance.